k8s/agent-resources-satellite-debug.yaml

```diff
@@ -0,0 +1,202 @@
+---
```

Comment:
wrt all 3 of these manifests, are we planning to distribute these on the cdn or are they just here for testing?

Reply:
probably not, instead document new params with examples. these are temps.